brands/amenity/fast_food.json

```diff
@@ -1176,6 +1176,23 @@
       "takeaway": "yes"
     }
   },
+  "amenity/fast_food|Meson Sandwiches": {
+    "countryCodes": ["us"],
+    "matchNames": [
+      "el meson",
```

Comment:
Most existing features in Puerto Rico are tagged “El Mesón”, which this line will match.